onnxruntime/python/onnxruntime_pybind_ortvalue.cc

```diff
@@ -15,7 +15,7 @@
 #include "core/framework/tensor.h"
 #include "core/framework/sparse_tensor.h"
 #include "core/framework/TensorSeq.h"
-#ifdef ENABLE_TRAINING 
+#ifdef ENABLE_TRAINING
```

Comment:
https://github.com/microsoft/onnxruntime/pull/11597/ will change several macros, rebase carefully when that one is merged

Reply:
Is it possible to merge this one before? It was started more than a month ago.

Reply:
@iK1D what do you say?

Reply:
After the discussion I will add a new macro named ENABLE_ATEN for ATen related thing including DLPack. But I guess it's OK to merge this before that.

Reply:
Thanks for the feedback. Who should finally review approve the changes?

Reply:
@thiagocrepaldi could you do a final review on the PR?